mne/epochs.py

```diff
@@ -394,6 +394,7 @@ def load_data(self):
         self._decim = 1
         self._raw_times = self.times
         assert self._data.shape[-1] == len(self.times)
+        self._raw = None  # shouldn't need it anymore
```

Comment:
yes, but I was thinking weather `preload=True` also calls `load_data()`. Should it now work for both cases `preload=True` and `epochs.load_data()`?